tic-tac-toe/src/tic_tac_toe/board.clj

```diff
@@ -0,0 +1,79 @@
+(ns tic-tac-toe.board
+  (:require [tic-tac-toe.marks :refer :all]))
+
+(def dimension 3)
+
+(defn- one-based [value]
+  (+ value 1))
+
+(defn- three-matching-player-symbols? [row]
+  (let [[one two three] row]
+    (and (= one two three) (not(= one nil)))))
+
+(defn- winning-row? [rows]
+  (some true? (map
+                (fn[row] (three-matching-player-symbols? row))
```

Comment:
Absolutely correct: when you leave the brackets out, the function itself is passed as an argument and executed in `map`